package.json

```diff
@@ -26,12 +26,12 @@
   },
   "devDependencies": {
     "@types/lunr": "^2.3.4",
+    "@types/jest": "^26.0.20",
     "lerna": "3.22.0",
     "ts-node": "9.0.0",
     "typedoc": "^0.20.36"
   },
   "dependencies": {
-    "@types/jest": "^26.0.20",
     "jest": "^26.6.3",
     "ts-jest": "^26.5.1",
```

Comment:
I think we can add these dependencies into `devDependencies` as we only need while we are developing.